homeassistant/components/stream/core.py

```diff
@@ -386,17 +386,27 @@ async def handle(
         raise NotImplementedError()
 
 
-TRANSFORM_IMAGE_FUNCTION = (
-    lambda image: image,  # Unused
-    lambda image: image,  # No transform
-    lambda image: np.fliplr(image).copy(),  # Mirror
-    lambda image: np.rot90(image, 2).copy(),  # Rotate 180
-    lambda image: np.flipud(image).copy(),  # Flip
-    lambda image: np.flipud(np.rot90(image)).copy(),  # Rotate left and flip
-    lambda image: np.rot90(image).copy(),  # Rotate left
-    lambda image: np.flipud(np.rot90(image, -1)).copy(),  # Rotate right and flip
-    lambda image: np.rot90(image, -1).copy(),  # Rotate right
-)
+def _transform_image(image: np.ndarray, orientation: int) -> np.ndarray:
+    """Transform the image using numpy."""
+
+    # Keep import here so that we can import stream integration without installing reqs
```

Comment:
```suggestion
    if orientation == 1:
        return image  # No transform
    # Keep import here so that we can import stream integration without installing reqs
```

Reply:
This feature is probably not used often (default orientation is 1), so we can avoid the import for the base case.